### ISSUE TITLE
imports inside raw string literals are removed

### ISSUE THREAD

**Issue by ekle** [rinchsan/gosimports#109]
```
package main

import (
	"fmt"
)

func main() {
	fmt.Println("hello world")
}

const helloWorld = `
package main

import (
	"fmt"
)

func main() {
	fmt.Println("hello world")
}
```
gets reformated to 
```
package main

import (
	"fmt"
)

func main() {
	fmt.Println("hello world")
}

const helloWorld = `
package main

import (
)

func main() {
	fmt.Println("hello world")
}
```
There the import is now broken.

**Comment by rinchsan** [comment]
@ekle 
Thank you for reporting a bug.
It was fixed by #113 and a new version including this fix has been released.
https://github.com/rinchsan/gosimports/releases/tag/v0.3.1

Sorry for your incovenience!

**Comment by ekle** [comment]
@rinchsan
No need to be sorry.
Thank you for fixing it!
